learning/dashboard/webserver/readme.md

```diff
@@ -0,0 +1,161 @@
+               http://servermania_server_ip
+               
+Step 3: Use systemctl to start, stop, or restart your Nginx  server:
+sudo systemctl stop nginxsudo systemctl start   nginxsudo systemctl restart nginx
+
+Step 4: The Nginx configuration file is found at /etc/nginx/nginx.conf. If you make changes to the configuration, you can either restart the Nginx server or reload the configuration file.
+          systemctl reload nginx # reloads Nginx configuration
+          
+Use the reload command rather than the restart command whenever possible because it doesn't affect any current connections you may have to Nginx.
+
+##	Configure Server Blocks
+
+Similar capability is offered by server blocks and the virtual hosts of Apache Web Server. The most widely utilised server in the LAMP Stack is Apache Web Server. See How to Easily Install Your Own Server for instructions on installing the LAMP stack on ServerMania servers.
+
+You can use more than one domain with your Nginx server thanks to Nginx server blocks. The steps below must be followed in order to set up two domains, domainone.com and domaintwo.com, on the same Nginx server:
+
+Create a directory, one for each domain:
+
+Nginx only creates one server block by default, and it points to the /var/www/html directory. If you simply had one site, this would function flawlessly. All of the website's content is housed in the html subdirectory. For both of your domains, create the html subdirectories.
+
+sudo mkdir -p /var/www/domainone.com/htmlsudo mkdir -p /var/www/domaintwo.com/html
+
+Step 1: Change ownership of the directories:
+Doing this will enable the user to create and edit the contents of these new directories.
+sudo chown -R $user:$user /var/www/domainone.com/htmlsudo chown -R $user:$user /var/www/domaintwo.com/html
+
+Step 2: Modify permissions for both the domain directories:
+This step is important to make sure that you don’t need root privileges to make changes to any of the sites.
+sudo chmod -R 755 /var/www
+
+Step 3: Create sample web pages for each domain/site:
+Create a landing page index.html for both domains. Create a basic HTML snippet for a welcome page for both sites.
+nano /var/www/domainone.com/html/index.html
+Once the editor is open, paste the following into the editor:
+<html> <head> <title>Welcome to ServerMania’s Domain One!</title> </head> <body> <h1>You have reached Nginx via Server Block #1</h1> </body></html>
+Save the index.html file and create a similar one for the second domain. Make sure you change the welcome message.
+
+Step 4: Create Server Blocks:
+As mentioned earlier, by default, Nginx has only one server block. You can find it here — /etc/nginx/sites-available/default. So, we'll need two server blocks because we have two sites to support. The server restricts access to port 80. For our new server blocks, let's replicate the default server block.
```

Comment:
Consider using a code block for commands and code snippets.
You can also use single line code snippets for file paths.